platforms/software/dependency-management/src/main/java/org/gradle/internal/locking/InvalidLockFileException.java

```diff
@@ -24,12 +24,12 @@
 public class InvalidLockFileException extends RuntimeException implements ResolutionProvider {
     private final String resolutionMessage;
 
-    public InvalidLockFileException(String name, Exception cause, String resolutionMessage) {
-        super("Invalid lock state for " + name, cause);
+    public InvalidLockFileException(String displayName, Exception cause, String resolutionMessage) {
```

Comment:
💭 Here, too, it's the lock id that matters, that should be in the message as well.

Reply:
This is a user-facing message, IMO it should use the display name of the thing getting locked -- which should point the user to the thing they want to fix. The lock ID is a unique ID for the thing being resolved, but I would not expect it to always be as pretty for human readable message. 

The difference would be

`Invalid lock state for configuration 'foo'`
vs
`Invalid lock state for 'foo'` 

Unless we want to put both the display name and lock ID in the message

Reply:
I was imagining both, for when it's no longer true that configurations are the only thing locked.

`Invalid lock state for configuration 'foo' (id: 'foo').` or something similar.  Redundant for now, but better for the future.